docs/running-on-yarn.md

```diff
@@ -492,6 +492,14 @@ To use a custom metrics.properties for the application master and executors, upd
   If it is not set then the YARN application ID is used.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.shuffle.service.name</code></td>
+  <td><code>spark_shuffle</code></td>
+  <td>
+    The name of the external shuffle service.
+    The external shuffle service itself is configured and started by YARN (see [Configuring the External Shuffle Service](#configuring-the-external-shuffle-service) for details). The name specified here must match the name used in YARN service implementation.
```

Comment:
I think it would help to mention that must match the name given to the shuffle service in `yarn-site.xml`  under `yarn.nodemanager.aux-services`.